sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -8,8 +8,11 @@
 
 #### Bugs Fixed
 * Fixed issue where Query Change Feed did not return items if the container uses legacy Hash V1 Partition Keys. This also fixes issues with not being able to change feed query for Specific Partition Key Values for HPK. See [PR 41270](https://github.com/Azure/azure-sdk-for-python/pull/41270/)
+* Fixed session container compound session token logic. The SDK will now only send the relevant tokens for each read request, as opposed to the entire compound session token for the container. See [PR 40366](https://github.com/Azure/azure-sdk-for-python/pull/40366).
+* Write requests will no longer send session tokens when using session consistency. See [PR 40366](https://github.com/Azure/azure-sdk-for-python/pull/40366).
```

Comment:
This isn't true for multi-writer accounts.

Reply:
right good call out